core/src/main/java/com/datastax/oss/driver/api/core/cql/TraceEvent.java

```diff
@@ -15,18 +15,21 @@
  */
 package com.datastax.oss.driver.api.core.cql;
 
+import edu.umd.cs.findbugs.annotations.Nullable;
 import java.net.InetAddress;
 
 /** An event in a {@link QueryTrace}. */
 public interface TraceEvent {
 
   /** Which activity this event corresponds to. */
+  @Nullable
   String getActivity();
```

Comment:
This is read directly from Cassandra tables. In theory it shouldn't be null, but the data could be corrupted (it's possible to alter it manually), so I agree with marking it nullable.